front/config.form.php

```diff
@@ -1,33 +1,31 @@
 <?php
-/**
- -------------------------------------------------------------------------
-
- LICENSE
-
- This file is part of Behaviors plugin for GLPI.
-
- PDF is free software: you can redistribute it and/or modify
- it under the terms of the GNU Affero General Public License as published by
- the Free Software Foundation, either version 3 of the License, or
- (at your option) any later version.
 
- PDF is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- GNU Affero General Public License for more details.
-
- You should have received a copy of the GNU Affero General Public License
- along with Reports. If not, see <http://www.gnu.org/licenses/>.
-
- @package   pdf
- @authors   Nelly Mahu-Lasson, Remi Collet
- @copyright Copyright (c) 2019-2022 PDF plugin team
- @license   AGPL License 3.0 or (at your option) any later version
-            http://www.gnu.org/licenses/agpl-3.0-standalone.html
- @link      https://forge.glpi-project.org/projects/pdf
- @link      http://www.glpi-project.org/
- @since     2009
- --------------------------------------------------------------------------
+/**
+ *  * @version $Id: HEADER 15930 2011-10-25 10:47:55Z jmd $
+ *  -------------------------------------------------------------------------
+ *  pdf - Export to PDF plugin for GLPI
+ *  Copyright (C) 2003-2011 by the pdf Development Team.
+ *
+ *  https://forge.indepnet.net/projects/pdf
+ *  -------------------------------------------------------------------------
+ *
+ *  LICENSE
+ *
+ *  This file is part of pdf.
+ *
+ *  pdf is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  pdf is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with pdf. If not, see <http://www.gnu.org/licenses/>.
+ *  --------------------------------------------------------------------------
```

Comment:
That's not only the author, the license has been changed also (from agpl to gpl), it should not.

Also, the first line of the new header is something that was used in subversion repositories, I guess that's no longer useful :D
forge link are dead, you probably can replace it with the correct one.

Reply:
If this should suffice (keep the original and adapt it) :
* add `Teclib` to authors
* Update forge link to github repo link

```php
/**
* -------------------------------------------------------------------------
* LICENSE
*
* This file is part of PDF plugin for GLPI.
*
* PDF is free software: you can redistribute it and/or modify
* it under the terms of the GNU Affero General Public License as published by
* the Free Software Foundation, either version 3 of the License, or
* (at your option) any later version.
*
* PDF is distributed in the hope that it will be useful,
* but WITHOUT ANY WARRANTY; without even the implied warranty of
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
* GNU Affero General Public License for more details.
*
* You should have received a copy of the GNU Affero General Public License
* along with Reports. If not, see <http://www.gnu.org/licenses/>.
*
* @package   pdf
* @author   Nelly Mahu-Lasson, Remi Collet, Teclib
* @copyright Copyright (c) 2009-2022 PDF plugin team
* @license   AGPL License 3.0 or (at your option) any later version
*            http://www.gnu.org/licenses/agpl-3.0-standalone.html
* @link      https://github.com/pluginsGLPI/pdf/
* @link      http://www.glpi-project.org/
* @since     2009
* --------------------------------------------------------------------------
*/
```

Reply:
@RomainLvr 
can you use this ☝️for all files
and also update `setup.php` -> `plugin_version_pdf()` to update `author` section

`'author'         => 'Remi Collet, Nelly Mahu-Lasson, Teclib',`